engine/access/rpc/backend/backend_scripts.go

```diff
@@ -378,9 +378,6 @@ func (b *backendScripts) tryExecuteScriptOnArchiveNode(
 	}(closer)
 	resp, err := archiveClient.ExecuteScriptAtBlockID(ctx, req)
 	if err != nil {
-		if status.Code(err) == codes.Unavailable {
-			b.connFactory.InvalidateAccessAPIClient(executorAddress)
-		}
```

Comment:
`archiveClient` is using the connection manager, so this invalidation will happen automatically